### PR TITLE
feat(overflow-menu): add size variants

### PR DESCRIPTION
**Pull request by kennylam**
### Related Ticket(s)

#533 

### Description

This adds size variants to `overflow-menu`.

### Changelog

**New**

- `sm` and `xl` sizes

### CODE REVIEW

File: src/components/overflow-menu/overflow-menu.scss [hunk 1]
```diff
@@ -13,6 +13,16 @@ $css--plex: true !default;
   @extend .#{$prefix}--overflow-menu;
 }
 
+:host(#{$prefix}-overflow-menu)[size='sm'] {
+  width: rem(32px);
+  height: rem(32px);
```
Comment (asudoh) [inline src/components/overflow-menu/overflow-menu.scss:18]:
```suggestion
  @extend .#{$prefix}--overflow-menu--sm;
```

File: src/components/overflow-menu/overflow-menu.scss [hunk 2]
```diff
@@ -13,6 +13,16 @@ $css--plex: true !default;
...
+}
+
+:host(#{$prefix}-overflow-menu)[size='xl'] {
+  width: rem(48px);
+  height: rem(48px);
```
Comment (asudoh) [inline src/components/overflow-menu/overflow-menu.scss:23]:
```suggestion
  @extend .#{$prefix}--overflow-menu--xl;
```

**Review by asudoh — COMMENTED** [review]
Thank you for jumping in @kennylam! Please let me know the change suggestion doesn't work.

File: src/components/overflow-menu/overflow-menu.scss (hunk 1 above)
Comment (kennylam) [reply src/components/overflow-menu/overflow-menu.scss]: @asudoh Good call! Interesting that this works, in the past I've had problems extending classes directly from other imports.

**Review by asudoh — APPROVED** [review]
Thanks @kennylam for the update!

**Pull request merged**